src/error.js

```diff
@@ -153,6 +153,17 @@ export function cancellation() {
 }
 
 /**
+ * @param {*} errorOrMessage
+ * @return {boolean}
+ */
+export function isCancellation(errorOrMessage) {
+  if (!errorOrMessage) {
+    return false;
+  }
+  return errorOrMessage == CANCELLED || errorOrMessage.message == CANCELLED;
```

Comment:
Errors are freely mutated while passing via error pipelines. E.g. errors are rewritten, etc. So, I think it'd be unrealistic to have a global const. Also, cancellations are rather rare.